wagtail/admin/models.py

```diff
@@ -1,5 +1,55 @@
+from django.contrib.contenttypes.models import ContentType
+from django.db.models import Count
+
+from modelcluster.fields import ParentalKey
+from taggit.models import Tag
+
 # The edit_handlers module extends Page with some additional attributes required by
 # wagtailadmin (namely, base_form_class and get_edit_handler). Importing this within
 # wagtailadmin.models ensures that this happens in advance of running wagtailadmin's
```

Comment:
Yep, I guess this was missed in the mass-rename for 2.0 because it didn't give the full module path... Fixed in 9a7d098049e9c8a986db53dc933c95e1540c6f5a.